node_modules/oae-core/activity/activity.html

```diff
@@ -65,7 +65,7 @@ <h4 class="media-heading">
                                     {else}
                                         ${item.comment.author.displayName|encodeForHTML}
                                     {/if}
-                                    <small class="muted"><time datetime="${item.comment.published}"></time></small>
+                                    <small class="muted"><time aria-live="off" datetime="${item.comment.published}"></time></small>
```

Comment:
Rather than setting `aria-live="off"` in all `time` instances that use timago, we could just add it to `var timeAgo = exports.timeAgo = function($container) {` in the l10n api, just before `$timeEl.timeago();`